core/capabilities/ccip/launcher/launcher.go

```diff
@@ -189,195 +191,180 @@ func (l *launcher) processDiff(diff diffResult) error {
 	return err
 }
 
+// processUpdate will manage when configurations of an existing don are updated
+// If new oracles are needed, they are created and started. Old oens wil
```

Comment:
Seems like the comment is unfinished

Reply:
I can change this when refreshing the go.mod of the chainlink-ccip version